src/sentry/remote_subscriptions/consumers/result_consumer.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+import abc
+import logging
+from collections.abc import Mapping
+from typing import Generic, TypeVar
+
+from arroyo.backends.kafka.consumer import KafkaPayload
+from arroyo.processing.strategies.abstract import ProcessingStrategy, ProcessingStrategyFactory
+from arroyo.processing.strategies.commit import CommitOffsets
+from arroyo.processing.strategies.run_task import RunTask
+from arroyo.types import BrokerValue, Commit, FilteredPayload, Message, Partition
+from sentry_kafka_schemas.codecs import Codec
+from sentry_kafka_schemas.schema_types.uptime_results_v1 import CheckResult
+
+from sentry.conf.types.kafka_definition import Topic, get_topic_codec
+from sentry.db.models import FlexibleForeignKey, Model
+from sentry.remote_subscriptions.models import RemoteSubscription
+
+logger = logging.getLogger(__name__)
+
+UPTIME_RESULTS_CODEC: Codec[CheckResult] = get_topic_codec(Topic.UPTIME_RESULTS)
```

Comment:
Why does this live here now?

Reply:
Oh I meant to totally remove this, it's unused